README.md

```diff
@@ -697,10 +733,12 @@ The `delete` subcommand deletes both the configuration file and its backup copy.
 The `pgenv remove` command also deletes any configuration for the removed
 version.
 
+
```

Comment:
What's with all the additional extra blank lines? (Style difference, not substantive, just curious.)

Reply:
I tend to be "large" on markdown, because my experience is that a single newline is sometime not understand as you expect.
However, removed in de7d6c3

Reply:
Huh. How have you seen them misunderstood, just out of curiosity?

Reply:
line breaks were not honoured. If I remember correctly, GH pages are the most evident example of this.